develop/tutorials/articles/110-portlets/07-using-javascript-in-your-portlets/03-using-npm-in-your-portlets/02-configuring-liferay-npm-bundler.markdown

```diff
@@ -46,8 +46,12 @@ Here's an example `.npmbundlerrc` configuration:
             (same as wildcard format (`*`) contents below)
         ]
       },
+      "include-dependencies": {
+          "some-package","another-package"...
+      },
       "output":(relative path to output directory),
       "process-serially":(true|false),
+      "verbose": <true|false>
```

Comment:
@izaera Minor SF question. Should there be a comma after the `verbose` configuration before the wildcard?

Reply:
Yes 👍

Reply:
Thanks!